setup.cfg

```diff
@@ -29,7 +29,7 @@ python_requires = >=3.7
 install_requires =
     inflection
     factory_boy>=2.10.0
-    pytest>=4.6
+    pytest>=6.0
```

Comment:
any reason why we can't keep supporting 4.6? I would drop compatibility only if it's in our way, otherwise what's the problem?

Reply:
Most pytest plugins drop support for `pytest~=4.6`, because [it's not officially maintained by the pytest team](https://docs.pytest.org/en/latest/py27-py34-deprecation.html#maintenance-of-4-6-x-versions) since mid-2020.
A lot of things changed in pytest since version 4.6.x, it's also probably not so widely used now as 1.5-2 years ago, so probably migration to a newer version will be required only by quite a small amount of users.

Reply:
Yep, the majority of PyPI downloads for pytest are for version 6 and 7.

pytest 5 is quite low, and 4.6 even lower:

<img width="880" alt="image" src="https://user-images.githubusercontent.com/1324225/166227160-44b5a53c-0bf8-480c-b398-72a028c8db77.png">

[https://pepy.tech/project/pytest?versions=4.6.*&versions=5.*&versions=6.*&versions=7.\*](https://pepy.tech/project/pytest?versions=4.6.*&versions=5.*&versions=6.*&versions=7.*)

Reply:
I want to at least keep support for pytest >=5 though. I don't see the point in forcing users to upgrade pytest to use our plugin without a reason.

Reply:
Personally, I think it's good to kinda "enforce" users to migrate to the newest library, but of course, I understand your motivations, so I will update the supported `pytest` version to `>=5.0`